Wireframe/css/style.css

```diff
@@ -0,0 +1,97 @@
+
+body {
+  font-size: 20px;
+  color: black;
+  margin-right: auto;
+  margin-left: 100px;
+  margin-right: 100px;
+  margin-bottom: 0;
+  background: #f0efe7;
+  padding: 0;
+  height: 100%;
+}
+
+header{
+  text-align: center;
+  margin-bottom: 5px;
+ 
+}
+h1,h2{
+  margin:5px;
+ 
+}
+article{
+  padding-left: 20px;
+}
+
+article:first-child {
+  margin-bottom: 10px;
+  border: 1px solid white;
+  border-radius: 15px;
+  height: auto;
+}
+
+#article-one-h2-margin{
+  margin-left: 80px;
+
+}
+#article-first-child-title{
+  display: flex;
+}
+
+#other-article-title{
+  display: flex;
+  justify-content: space-between;
+  margin-left: 100px;
+  width: 100%;
+  height: 500px;
+  margin:5px;
+  border: 1px solid white;
+  border-radius: 15px;
```

Comment:
Border and border-radius makes it look a bit odd. I would suggest to remove it but wanna know what is the reason behind adding it?

Reply:
**Border-radius:**
I have removed the border radius since the edge of the wireframe is square-shaped. However, I use it to create a round effect at the edge of the border.
**Border:**
I retained the border to create a style effect on the page. The text will appear to be on a frame or canvas. The colour is now very similar to the background colour hoping that removes the skeletal line that was making it look odd. 
Overall, using the border helps when troubleshooting with dev tools(Inspect in Chrome)